modules/nodes-cluster-overcommit-node-enforcing.adoc

```diff
@@ -72,12 +72,11 @@ spec:
     matchLabels:
       pools.operator.machineconfiguration.openshift.io/worker: "" <2>
   kubeletConfig:
-    cpuCfsQuota: <3>
-      - "true"
+    cpuCfsQuota: false <3>
```

Comment:
@xenolinux . Change LGTM and the preview looks good. I just want to confirm that dropping the quotation marks is correct?

"true" versus false.

@mburke5678 . Would you be OK to validate my peer review?

Reply:
@sunilcio Can you please help me with this question? https://github.com/openshift/openshift-docs/pull/63734#discussion_r1307417222

Reply:
> @xenolinux . Change LGTM and the preview looks good. I just want to confirm that dropping the quotation marks is correct?
> 
> "true" versus false.

Confirmed: It is correct